thinc/loss.py

```diff
@@ -46,11 +46,14 @@ def __init__(
         names: Optional[List[str]] = None,
         missing_value: Optional[Union[str, int]] = None,
         neg_prefix: Optional[str] = None,
+        alpha: float = 0.0
```

Comment:
`alpha` is not very descriptive. Maybe `label_smoothing` or `label_smoothing_prob`?

Reply:
`smooth_prob`?

Reply:
It's called `label_smoothing` in Tensorflow/Keras. If we need something shorter, I like `smoothing`.

Reply:
I guess I would intuitively call this `epsilon`, but we should probably follow whatever convention is most used.

Reply:
And `epsilon` invokes other meanings when thinking about losses and comparing float values...

Reply:
(just rambling out loud here)